include/seastar/core/timer-set.hh

```diff
@@ -151,15 +160,28 @@ public:
      * Postconditions:
      *  - timer is no longer in the active set.
      *  - this object will no longer hold any references to this timer.
+     *
+     * get_next_timeout() will return correct next time point after removing
+     * timer with earliest timeout.
```

Comment:
What was "incorrect" about the currently returned value? 

The doc about get_next_timeout() says:
```
    /**
     * Returns a time point at which expire() should be called
     * in order to ensure timers are expired in a timely manner.
     *
     * Returned values are monotonically increasing.
     */
```
So it is the time to call next expire(), not the time of the nearest expiring timer. After you call expire(), next timeout will be recalculated based on currently active timers.

Reply:
removed all comment